assets/app/view/game/route_selector.rb

```diff
@@ -183,23 +189,35 @@ def render
           },
         }
 
-        instructions = 'Click revenue centers, again to cycle paths.'
-        instructions += ' Click button under Revenue to pick number of halts.' if render_halts
+        if @step.respond_to?(:instructions)
```

Comment:
avoid doing respond_to, please remove all of them and add implementation in base game

Reply:
Unfortunately, this has be specific to a step, not a game, since I need route_selector to work normally for the actual run_routes step.